### PR TITLE
shadow gradle plugin removed

### PR DESCRIPTION
**Pull request by bgalek**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -91,26 +88,10 @@ dependencies {
...
-    prefix = "axion"
-}
-
-tasks.shadowJar.dependsOn tasks.relocateShadowJar
-
-shadowJar {
-    archiveClassifier.set('')
-}
-
 components.java.withVariantsFromConfiguration(configurations.runtimeElements) {
```
Comment (cloudshiftchris) [inline build.gradle:95]: This section is only for shadow, removing dependencies from POM for publishing (as they are included in the shadow jar).  Should be removed to publish a full POM with dependencies.

File: build.gradle [hunk 2]
```diff
@@ -57,8 +54,8 @@ sourceSets {
 }
 
 dependencies {
-    shadow gradleApi()
-    shadow localGroovy()
+    implementation gradleApi()
```
Comment (cloudshiftchris) [inline build.gradle:57]: gradleApi() is automatically added by java-gradle-plugin, and not required here.  If we do with to leave it, it should be 'compileOnly' as the target gradle install (where the plugin is applied) provides this.

File: build.gradle [hunk 3]
```diff
@@ -57,8 +54,8 @@ sourceSets {
...
+    implementation localGroovy()
```
Comment (cloudshiftchris) [inline build.gradle:58]: localGroovy() should be in compileOnly, as Groovy is provided by Gradle as the plugin is applied.

File: build.gradle (hunk 1 above)
Comment (bgalek) [reply build.gradle:95]: yes, I've just tested this pr and found this out ;)

File: build.gradle (hunk 2 above)
Comment (bgalek) [reply build.gradle:57]: thanks for pointing this out!

**Pull request merged**